gpustack/worker/backends/vllm.py

```diff
@@ -182,13 +182,29 @@ def _get_configured_env(self, is_distributed: bool) -> Dict[str, str]:
         # Apply GPUStack's inference environment setup
         env = super()._get_configured_env()
 
+        # Optimize environment variables
+        # -- Disable OpenMP parallelism to avoid resource contention, increases model loading.
+        env["OMP_NUM_THREADS"] = env.pop("OMP_NUM_THREADS", "1")
```

Comment:
Should we put them before `env = super()._get_configured_env()`? Looks like users will not be able to override them